### PR TITLE
Add create directory button

### LINKED ISSUE #19: Implement 'Create new directory' action in kebab

### PR DESCRIPTION
**Pull request by MahmoudHamdy02**
Closes #19.

* [x] Correctly align form
* [x] Add tests

### CODE REVIEW

**Comment by MahmoudHamdy02** [comment]
I have tried adding `Flex` and `FlexItem` but that makes the text input have a weird smaller size

File: src/app.jsx [hunk 1]
```diff
@@ -391,6 +396,9 @@ const DropdownWithKebab = ({ selected, path, setPath, setPathIndex }) => {
                 </MenuToggle>}
         >
             <DropdownList>
+                <DropdownItem id="create-item" itemId='create-item' key="create-item" onClick={createDirectory}>
```
Comment (KKoukiou) [inline src/app.jsx:399]: itemId does  not seem to be a documented property for dropdown  item. https://patternfly-react-v5.surge.sh/components/menus/dropdown/#dropdownitem

Can you remove this from this and the delete entry just bellow?

File: src/app.jsx [hunk 2]
```diff
@@ -467,3 +475,39 @@ const ForceDeleteModal = ({ selected, itemPath, errorMessage, deleteFailed }) =>
...
+            isOpen
+            onClose={Dialogs.close}
+            footer={errorMessage === undefined &&
+                <>
+                    <Button variant='primary' onClick={createDirectory}>{_("Create")}</Button>
+                    <Button variant='link' onClick={Dialogs.close}>{_("Cancel")}</Button>
+                </>}
+        >
+            {errorMessage === undefined &&
+                <>
```
Comment (KKoukiou) [inline src/app.jsx:501]: You need to use Form element from patternfly.

Comment (KKoukiou) [reply src/app.jsx:501]: And then FormGroup for displaying the label, field.

Comment (KKoukiou) [reply src/app.jsx:501]: Plus you should always show the form contents, now only when there is no error.

**Review by KKoukiou — CHANGES_REQUESTED** [review]

File: src/app.jsx (hunk 2 above)
Comment (MahmoudHamdy02) [reply src/app.jsx]: Updated! The error could use some better spacing though
image

Comment (KKoukiou) [reply src/app.jsx]: The error should appear on top of the form. You can use Stack layout for spacing

**Review by KKoukiou — APPROVED** [review]

**Pull request merged**